ContentfulからのWebhookでビルドに失敗する

こんなエラーが出たので直す
```bash
16:01:01 ▶ src/pages/blogs/entries/[entry_id].astro
Error: 1 [ERROR] [build] Failed to call getStaticPaths for src/pages/blogs/entries/[entry_id].astro
{
  "status": 401,
  "statusText": "Unauthorized",
  "message": "The access token you sent could not be found or is invalid.",
  "details": {},
  "request": {
    "url": "https://preview.contentful.com:443/spaces/***/environments/master/entries",
    "headers": {
      "Accept": "application/json, text/plain, */*",
      "Content-Type": "application/vnd.contentful.delivery.v1+json",
      "X-Contentful-User-Agent": "sdk contentful.js/11.2.0; platform node.js/v22.9.0; os Linux/v22.9.0;",
      "Authorization": "***",
      "User-Agent": "axios/1.7.7",
      "Accept-Encoding": "gzip, compress, deflate, br"
    },
    "method": "get"
  },
  "requestId": "a6992067-7bb8-4f00-915e-063[46](https://github.com/inajika/my-homepage/actions/runs/11640761011/job/32418480065#step:5:47)74bad5f"
}
  Stack trace:
    at errorHandler (/home/runner/work/my-homepage/my-homepage/node_modules/contentful-sdk-core/dist/index.js:1086:15)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async Module.getStaticPaths (file:///home/runner/work/my-homepage/my-homepage/dist/pages/blogs/entries/_entry_id_.astro.mjs?time=1730[53](https://github.com/inajika/my-homepage/actions/runs/11640761011/job/32418480065#step:5:54)0861712:13:23)
    at async getPathsForRoute (file:///home/runner/work/my-homepage/my-homepage/node_modules/astro/dist/core/build/generate.js:201:25)
    at async generatePages (file:///home/runner/work/my-homepage/my-homepage/node_modules/astro/dist/core/build/generate.js:103:7)
Error: Process completed with exit code 1.
```

_Originally posted by @inajika in https://github.com/inajika/my-homepage/issues/51#issuecomment-2452900252_

--- a/.github/workflows/publish-blog-entry.yaml
+++ b/.github/workflows/publish-blog-entry.yaml
@@ -6,7 +6,7 @@ on:
 
 env:
   TZ: Asia/Tokyo
-  NODE_ENV: development
+  NODE_ENV: production
   CONTENTFUL_SPACE_ID: ${{ secrets.CONTENTFUL_SPACE_ID }}
   CONTENTFUL_TOKEN: ${{ secrets.CONTENTFUL_TOKEN_PROD }}
 

